### ISSUE TITLE
Broken on 3.6

### ISSUE THREAD

**Issue by wumpus** [nathandines/SPF2IP#2]
The current code always returns an empty answer on Python 3.6. This fixes it for me:

```
diff --git a/SPF2IP.py b/SPF2IP.py
index e6210f3..84683ff 100644
--- a/SPF2IP.py
+++ b/SPF2IP.py
@@ -34,7 +34,7 @@ def dns_request_unicode(hostname,record_type,*args,**kwargs):
         value = value.__str__().encode('utf-8').decode('utf-8')
       output.append(value)
     elif record_type == "TXT":
-      value = ''.join([str(ent) for ent in entry.strings])
+      value = ''.join([str(ent, encoding='ascii') for ent in entry.strings])
       if type(value) is not unicode:
         value = value.decode('utf-8')
       output.append(value)
```
I only know python3 so I can't give good advice on making code that works on both 2 and 3. But a friend of mine's package has this function that you might find useful:

```
def to_native_str(value, encoding='utf-8'):
    if isinstance(value, str):
        return value

    if six.PY3 and isinstance(value, six.binary_type):  #pragma: no cover                                                                                      
        return value.decode(encoding)
    elif six.PY2 and isinstance(value, six.text_type):  #pragma: no cover                                                                                      
        return value.encode(encoding)
```

**Comment by nathandines** [comment]
Hey @wumpus, Thanks for raising this.

You might have seen that I've now resolved this in #3.

Unfortunately, the change which you'd proposed wasn't compatible with Python 2, but I've implemented a fix which should cover both versions.

**Comment by wumpus** [comment]
Thanks, glad that you're smarter about 2-to-3 issues than I am!
